### ISSUE TITLE
A11y_AzureDataStudio_Connect to a SQL Server_Keyboard: The New Connection tooltip is not accessible using keyboard

### ISSUE THREAD

**Issue by VijayaBharathyA** [microsoft/azuredatastudio#18570]
### GitHubTags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #BM-AzureDataStudio-Win32-Feb2022; #Azure Data Studio; #WCAG2.1.1; #Win32; #AINotInScope; #Keyboard;

### Environment Details:
Windows Version :Windows 11 Enterprise
Application name :Azure Data studio

### Repro Steps:
1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Hover over the 'Servers' header and select the 'New Connection' icon.
4. Verify tooltip is accessible using keyboard or not 

### Actual Result:
The New Connection tooltip is not accessible using keyboard

### Expected Result:
The New Connection tooltip should be accessible using keyboard

### Note:
Similar issue can be observed in throughout the application wherever icon have a tooltip

### Attachment:
![A11y_AzureDataStudio_Connect to a SQL Server](https://user-images.githubusercontent.com/100189407/155978470-9588c78a-d02a-4fb3-9f0c-cebbff5c0f7d.gif)

**Comment by alanrenmsft** [comment]
upstream issue, being tracked here: https://github.com/microsoft/vscode/issues/132344
